docs/client/GatewaysApi.md

```diff
@@ -0,0 +1,37 @@
+# pureport\client\GatewaysApi
+
+All URIs are relative to *https://api.pureport.com*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**GetGatewayBGPRoutes**](GatewaysApi.md#GetGatewayBGPRoutes) | **Get** /gateways/{gatewayId}/bgpRoutes | Get gateway bgp routes
+
+
+# **GetGatewayBGPRoutes**
+> []BgpRoute GetGatewayBGPRoutes(ctx, gatewayId)
+Get gateway bgp routes
+
+
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **gatewayId** | **string**|  | 
+
+### Return type
+
+[**[]BgpRoute**](BGPRoute.md)
+
+### Authorization
+
+No authorization required
```

Comment:
This isn't true, is it? @mtraynham

Reply:
The codegen is creating this since we don't define a `security` field in our Swagger Schema.
That's probably something we should add. I'll create a US for it.

Reply:
https://pureport.atlassian.net/browse/PE-1310